easybuild/easyconfigs/m/mDust/mDust-20231029-GCC-10.3.0.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'mDust'
+version = '20231029'
```

It's better and clearer to stick to the date of the used commit here.
```suggestion
version = '20150102'
```